spec/features/creating_peeps_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+feature "User creates new peep" do
+
+  scenario "when browing the homepage" do
+    expect(Peep.count).to be 0
+    visit '/'
+    post_peep("Hello Chitter", "Costas")
+    expect(Peep.count).to eq 1
+    peep = Peep.first
+    expect(peep.content).to eq "Hello Chitter"
+    expect(peep.user).to eq "Costas"
+  end
+
```

Comment:
multiple expectations
You could consider testing the database with unit tests, thus keeping the responsibility of the features to test the user experience - so for instance, instead of testing Peep.count to eq 1, test if you can see 1 peep on the page.